Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -224,19 +228,26 @@ namespace gdjs {
         logger.error('Missing game id in project properties.');
         return;
       }
-      const authenticatedUserStorageItem = window.localStorage.getItem(
-        getLocalStorageKey(gameId)
-      );
-      if (!authenticatedUserStorageItem) {
+      try {
+        const authenticatedUserStorageItem = window.localStorage.getItem(
+          getLocalStorageKey(gameId)
+        );
+        if (!authenticatedUserStorageItem) {
+          _checkedLocalStorage = true;
+          return;
+        }
+        const authenticatedUser = JSON.parse(authenticatedUserStorageItem);
+
+        _username = authenticatedUser.username;
+        _userId = authenticatedUser.userId;
+        _userToken = authenticatedUser.userToken;
         _checkedLocalStorage = true;
-        return;
+      } catch (err) {
+        logger.warn(
+          'Unable to read authentication details from localStorage. Player auth will not be available.',
```

Comment:
```suggestion
          'Unable to read authentication details from localStorage. Player authentication will not be available.',
```

(Remember that warnings are user visible and can be cryptic for a new game creator)